the-collab-lab/tcl-10-smart-shopping-list
# Main styling

## Description

- styled list component (items and title, not input) with material-ui
- Added share token display
- Styled footer navigational bar with material-ui
- Added `{Link}` from react-router to add the links to `List.js` and `AddItem.js`.
This code focused on styling rather than functionality.

## Related Issue

closes #41
closes #48 

## Acceptance Criteria
- list items styled with material-ui according based on [figma wireframes.](https://www.figma.com/file/MMUP1esyOfgOGX39UoXk8u/TCL---Group-10)



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |
| ✓ | :sparkles: Styling update     |

## Updates

### Before
![image](https://user-images.githubusercontent.com/50968795/93689255-c8439000-fa9a-11ea-8c5d-d7c1b2b4e4a1.png)

### After
![image](https://user-images.githubusercontent.com/50968795/93689259-cf6a9e00-fa9a-11ea-8337-c37a63216dd4.png)

Styled Navigation Bar
![Screen Shot 2020-09-18 at 12 21 55 PM](https://user-images.githubusercontent.com/43456792/93690955-e1990c00-fa9b-11ea-992b-b868da08e055.png)


## Testing Steps / QA Criteria
- using the token `deify chalk tecum`, check that all functionality still works as before
- create a new list and see that the share token changes accordingly.
- Click each icon to be directed to the page linked to that icon.
